nixos/modules/virtualisation/nova-config.nix

```diff
@@ -24,7 +24,7 @@ with lib;
     # Allow root logins
     services.openssh = {
       enable = true;
-      permitRootLogin = "prohibit-password";
+      permitRootLogin = "without-password";
```

Comment:
Now I remember. `prohibit-password` was introduced later in sshd as an less ambitious alias for `without-password`.
So both options have the same effect.